src/api/profile.ts

```diff
@@ -0,0 +1,15 @@
+export const getUsersData = async (id : any) => {
```

Comment:
I hate to be anal about naming but let's keep same naming conventions as the other api files and name this fetchUser or loadUser. Also let's rename the file to user.ts to reflect the table in the database